libraries/standard/mqtt/src/iot_mqtt_subscription.c

```diff
@@ -161,7 +161,7 @@ static bool _matchEndWildcards( const char * pTopicFilter,
     if( endChar == true )
     {
         /* Determine if the topic filter ends with the '#' wildcard. */
-        status = ( pTopicFilter[ filterIndex + 1U ] == '/' ) && ( pTopicFilter[ filterIndex + 2U ] == '#' );
```

Comment:
Is `pTopicFilter[filterIndex+1] == '/'` not checked due to being validated elsewhere?

Reply:
Correct